compiler/luci/pass/src/FusePreActivationBatchNormPass.cpp

```diff
@@ -466,10 +466,12 @@ bool fuse_mul_with_conv(luci::CircleMul *mul)
 
       // Update origin
       // TODO need to remove const
-      luci::add_origin(const_cast<luci::CircleConv2D *>(conv), luci::get_origin(mul));
+      luci::add_origin(const_cast<luci::CircleConv2D *>(conv),
+                       luci::get_origin(loco::must_cast<luci::CircleNode *>(s)));
```

Comment:
@jinevening Can you check this...?

Reply:
I think the previous code is right, because `mul` is fused to `conv`.

Reply:
I thought that ADD and MUL is fused to preceding CONV but it was not.
Thanks for checking!